### PR TITLE
Singularity

### PR DESCRIPTION
**Pull request by garvct**
- mv the examples/singularity dir to examples/simple_hpc_pbs_singularity
- Added PBS
- Created readme file.

### CODE REVIEW

File: examples/simple_hpc_pbs_singularity/readme.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Build a PBS compute cluster
+
+Visualisation: [config.json](https://azurehpc.azureedge.net/?o=https://raw.githubusercontent.com/Azure/azurehpc/master/examples/simple_hpc_pbs_singulraity/config.json)
```
Comment (xpillons) [inline examples/simple_hpc_pbs_singularity/readme.md]: Typo

File: examples/simple_hpc_pbs_singularity/config.json [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    "install_from": "headnode",
+    "admin_user": "hpcadmin",
+    "variables": {
+        "acr_repo": "<NOT-SET>",
+        "key_vault": "<NOT-SET>",
+        "monitor_workspace": "<NOT-SET>",
+        "hpc_image": "OpenLogic:CentOS-HPC:7.6:latest",
+        "location": "southcentralus",
+        "resource_group": "<NOT-SET>",
+        "vm_type": "Standard_HB60rs",
```
Comment (xpillons) [inline examples/simple_hpc_pbs_singularity/config.json]: Please don't use HB for Headnode, use DS_v3 instead. Add an additional VM type variable for the headnode as well as and additional image.

File: examples/simple_hpc_pbs_singularity/config.json [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+            "script": "singularity_setup.sh",
+            "sudo": true,
+            "tag": "container",
+            "args": [
+                "variables.acr_repo",
+                "acrkey.{{variables.acr_repo}}"
+            ]
+        },
+        {
+            "script": "analytics.sh",
```
Comment (xpillons) [inline examples/simple_hpc_pbs_singularity/config.json]: analytics.sh is not in the repo.

File: examples/simple_hpc_pbs_singularity/readme.md (hunk 1 above)
Comment (garvct) [reply examples/simple_hpc_pbs_singularity/readme.md]: Fixed.

File: examples/simple_hpc_pbs_singularity/config.json (hunk 2 above)
Comment (garvct) [reply examples/simple_hpc_pbs_singularity/config.json]: Changed.

**Pull request merged**